### ISSUE TITLE
md-slider is not dragable 

### ISSUE THREAD

**Issue by yelenka** [angular/material2#5128]
Hi all,
I have an issue with md-slider. Drag doesn’t work, though click on slider does set the value. I using the Angular 4.1.2, Material 2.0.0-beta.6. 
I’m doing the imports in this way: 
App.module.ts :
imports: [BrowserAnimationsModule, BrowserModule, HttpModule]
also I added import ‘hammerjs’; to the app.module
my.module.ts
imports: [MaterialModule, MdSliderModule]
Is there anything else I should know?

**Comment by willshowell** [comment]
Could you reproduce in a plunker? 

It _may_ be related to import ordering like this similar issue https://github.com/angular/material2/issues/4278 but it's hard to say based on your description.

**Comment by kara** [comment]
Closing this issue as it does not follow the issue template. Please file a new issue that follows the template, which gives the team the information needed to investigate.

**Comment by FlawaCLV** [comment]
This issue deserve to be taken seriously  _— template or not —_ as it renders a poor user experience.

**Comment by willshowell** [comment]
@FlawaCLV it's impossible to troubleshoot with such little information. If you can provide a reproduction and version information, the team will absolutely take it seriously!

**Comment by yelenka** [comment]
I provided all necessary information, versions of Angular and Materials, the order of imports in the app.module and the order of imports in my module, I cant provide the example in plunker because in plunker it works, but not in my app.
I no longer need your help, since I found the solution by my self.
Im using webpack so I change the order of loading the modules in that way:

'hammerjs',
'@angular/core',
'@angular/common',
'@angular/compiler',
'@angular/animations',
'@angular/animations/browser',
'@angular/http',
'@angular/forms',
'@angular/router',
'@angular/platform-browser',
'@angular/platform-browser/animations',
'@angular/platform-browser-dynamic',
'@angular/material',
......
and I have added those modules that are also needed : @angular/animations, @angular/animations/browser, @angular/platform-browser/animations

in my app root module the imports are in this order:
@NgModule({
    imports: [
        BrowserAnimationsModule,
        BrowserModule,
        MaterialModule,
        HttpModule,
        .........

The changes in webpack configuration that what helps me to resolve the issue.
